docs/how-to/rocks/add-internal-user-in-rocks.rst

```diff
@@ -0,0 +1,55 @@
+.. _add_internal_user_in_rocks:
+
+How to add an internal user to a rock
+*************************************
+
+Rockcraft allows you to use ``run-user`` to specify the user you want to run
+the rock's service(s) with. Without using that field in your rockcraft.yaml,
+the service(s) will be run by the root user. The downside, however, of using
+``run-user`` is that it only supports one user called ``_daemon_`` with a
+UID/GID of 84792. For most daemon services that's sufficient, but other
+services require that the running user should have a specific username and/or
+UID.
+
+In order to add a new internal user or group to a rock, two packages (or their
+respective slices) are needed:
+- ``base-files``: to create the base dirs such as ``/root`` and ``/home``,
+- ``base-passwd``: to produce basic ``/etc/passwd`` and ``/etc/group`` files.
+
+Invoking the ``useradd`` and ``groupadd`` commands can take place in the
+:ref:`build <lifecycle>` step, by appending the commands to the build flow.
+However, the changes made by those commands normally apply to the base system,
```

Comment:
This is not clear. The way I read it is:
"if i use `adduser` in the build step, they will affect my base system"

1- that's not the case
2- what's the "base system"?

Reply:
Reworded.